### ISSUE TITLE
Looking for ideas to have more 5V and Grow ports Day #14

### ISSUE THREAD

**Issue by WROMagnet** [WROMagnet/Magnet-Robots#91]
**Today, September 17, we will be talking in detail about the robot in the workshop to learn about other people's ideas and be able to choose the most effective one.**

- Today's Goal = The goal is to discuss as a group how we will add more 5V and Grow ports.

**Comment by WROMagnet** [comment]
### We have had several ideas to add more 5V and Grow ports, the ideas have been:

- Place a Protoboard and remove one of the Arduino UNOs.

- Place an adapter on the Arduino UNO that does not control the motors.

- Solder all the Grw and turn them into a cable and the same with the 5V.

**Comment by WROMagnet** [comment]
![Image](https://github.com/user-attachments/assets/c175c1f4-c3ff-4c07-9d65-e358bf143066)

We mainly started by placing a Protoboard as a test and removing one of the Arduino UNOs.

**Comment by WROMagnet** [comment]
We have noticed that this way the cables can come loose due to any movement or impact, since the cables we are using are much looser than normal.

I'm planning to try now to replace the Arduino UNO that was removed and put the adapter on it.

![Image](https://github.com/user-attachments/assets/d5e0aa19-5a96-4435-8cf3-764c6a813201)

This is the adapter I plan to use.

![Image](https://github.com/user-attachments/assets/1baf003f-2f68-4f96-bcb6-2ded4b261008)

- These are the 5V and Grw ports on this adapter. We'll plug it in and see if the 5V and Grw ports are enough to get everything running.

_With this I conclude this day._
